package.json

```diff
@@ -55,7 +55,7 @@
 		"del": "^3.0.0",
 		"fs-extra": "^5.0.0",
 		"js-yaml": "^3.10.0",
-		"marked": "^0.3.9",
+		"marked": "0.3.9",
```

Comment:
I'd change that actually to the latest 0.3.15 and fix the fixtures for emails but we can do that in a separate PR